docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.ipynb

```diff
@@ -0,0 +1,1087 @@
+   },
+   "source": [
+    "**Codeblock 27**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "2tHbd1x7MiiX",
+    "outputId": "007938cd-10ea-40d1-cd26-b6bc1f7b0ea6"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ZNE value: 0.9999972208991856\n"
+     ]
+    }
+   ],
+   "source": [
+    "@zne.zne_decorator(factory=zne.inference.ExpFactory(scale_factors=[1, 3, 5, 7]), scale_noise=zne.scaling.fold_gates_at_random)\n",
+    "def execute(circuit: cirq.Circuit) -> float:\n",
+    "    return cirq.DensityMatrixSimulator().simulate(\n",
+    "        circuit.with_noise(cirq.depolarize(p=0.01))\n",
+    "    ).final_density_matrix[0, 0].real\n",
+    "\n",
+    "\n",
+    "zne_value = execute(circuit)\n",
+    "print(\"ZNE value:\", zne_value)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "P5HEyCXwMio8"
+   },
+   "source": [
+    "**Codeblock 28**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "-AHS0pcLMixV",
+    "outputId": "0c26b4a3-77d0-4668-c799-efbd746f00fc"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ZNE then PEC value: -0.6000000797979894\n"
+     ]
+    }
+   ],
+   "source": [
+    "@pec.pec_decorator(representations=[h_rep], num_samples=10)\n",
+    "@zne.zne_decorator(\n",
+    "    factory=zne.inference.ExpFactory(scale_factors=[1, 3, 5, 7]), \n",
+    "    scale_noise=zne.scaling.fold_gates_at_random\n",
+    ")\n",
+    "def execute(circuit: cirq.Circuit) -> float:\n",
+    "    return cirq.DensityMatrixSimulator().simulate(\n",
+    "        circuit.with_noise(cirq.depolarize(p=0.01))\n",
+    "    ).final_density_matrix[0, 0].real\n",
+    "\n",
+    "\n",
+    "zne_then_pec_value = execute(circuit3)\n",
+    "print(\"ZNE then PEC value:\", zne_then_pec_value)  # Note this is not accurate (bad representation)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LQYPqUazMi3b"
+   },
+   "source": [
+    "**Codeblock 29**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "vWUzgR6EMi9n",
+    "outputId": "f2f56f66-9d37-4a17-f923-18c47a12ac5e"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "!pip install qiskit==0.24.0 --quiet\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "dyk7Hj1nPB48"
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "import qiskit\n",
+    "\n",
+    "\n",
+    "# provider = qiskit.IBMQ.load_account()  # Use this with saved credentials.\n",
+    "provider = qiskit.BasicAer  # Workaround.\n",
```

Comment:
I would just make this the default and just describe in a comment and just mention the local credentials (esp if mentioned in the paper)

Reply:
This has now been changed to make the simulator just the default.

Reply:
Better way to do this (from ibmq example):

```python
if qiskit.IBMQ.stored_account():
    provider = qiskit.IBMQ.load_account()
    backend = provider.get_backend("ibmq_qasm_simulator")  # Set quantum computer here!
else:
    # Default to a simulator.
    backend = qiskit.BasicAer.backends()[0]
```